box_embeddings/initializations/uniform_boxes.py

```diff
@@ -111,8 +111,9 @@ def __call__(self, t: torch.Tensor) -> None:  # type:ignore
         )
         with torch.no_grad():
             W = self.box_type_factory.box_subclass.W(z, Z, **self.box_type_factory.kwargs_dict)  # type: ignore
-
+            # print(W.shape,t.shape)
```

Comment:
Remove the comments with print statements

Reply:
Will do